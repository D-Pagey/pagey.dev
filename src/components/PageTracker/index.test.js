import React from 'react';
import { PageTracker } from '.';

describe('PageTracker component', () => {
    it('should render', () => {
        const { container } = render(<PageTracker />);
        expect(container.firstChild).toMatchSnapshot();
    });

    it.todo('should render returned github data');
});
