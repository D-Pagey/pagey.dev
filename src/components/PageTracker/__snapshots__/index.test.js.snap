// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTracker component should render 1`] = `
.c0 {
  margin: 0;
  padding: 0 0 2rem;
}

.c1 {
  display: grid;
  grid-gap: 2rem;
}

@media (min-width:768px) {
  .c1 {
    grid-template-columns: 1fr 1fr;
    grid-gap: 2rem;
  }
}

<div>
  <h1
    class="c0"
  >
    Github Tracking
  </h1>
  <div
    class="c1"
  />
</div>
`;
